O3-1.tex

```diff
@@ -319,8 +317,8 @@ \section{The LSST software stack}
 Public License, Version 3. The stack
 prototype code and documentation are available via \url{http://ls.st/ug}.
 Its open source nature, an open development
-process, attention to software engineering, a long-term project
-commitment and a modular design that can be modified for use with
+process, a long-term project
```

Comment:
The paragraph still states that the design can be modified for use with other cameras. I think the quibbling is over the definition of "modular".